ci/tasks/endtoend/card/docker-compose.yml

```diff
@@ -231,7 +231,7 @@ services:
       driver: "json-file"
 
   stubs_proxy:
-    image: ${DOCKER_REGISTRY_URI:-}${repo_reverse_proxy:-govukpay/reverse-proxy}:latest-master
+    image: ${DOCKER_REGISTRY_URI:-}${repo_reverse_proxy:-govukpay/reverse-proxy}:${tag_reverseproxy:-latest-master}
```

Comment:
Is this `stubs` or `reverse_proxy`?

Reply:
It's reverse proxy, notice this is the stubs_proxy app, not the stubs app :)

Reply:
Ah stubs is just above it - got it 👍